PluginsAndFeatures/azure-toolkit-for-intellij/resources/META-INF/plugin.xml

```diff
@@ -19,6 +19,28 @@
   <change-notes>
     <![CDATA[
     <html>
+    <h3>[3.14.0]</h3>
+      <h4>Added</h4>
+      <ul>
+        <li>Support to show application settings of Deployment Slot.</li>
```

Comment:
Support to show application settings **for** Deployment Slot.    of v.s. for  which one is better ?